addons/account_payment/account_payment.py

```diff
@@ -267,7 +267,10 @@ def _get_date(self, cr, uid, context=None):
 
     def _get_ml_inv_ref(self, cr, uid, ids, *a):
```

Comment:
Maybe better to just expand the signature to the standard signature for old API function fields: `function(self, cr, uid, ids, field, arg, context=None)`

Reply:
Great, thanks for the tip. That is indeed the cleanest way to do it.